### PR TITLE
Update ms-rest to latest

### PR DESCRIPTION
**Pull request by zolvarga**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: digitaltwins/model_repository/src/pl/models/index.ts [hunk 1]
```diff
@@ -1,308 +1,202 @@
...
-   * @member {string} [urnId]
-   */
   urnId?: string;
-  /**
-   * @member {string} [modelName]
-   */
   modelName?: string;
-  /**
-   * @member {number} [version]
-   */
```
Comment (jebrando) [inline digitaltwins/model_repository/src/pl/models/index.ts]: Did you mean to remove the comments?

**Comment by jebrando** [comment]
There is another reference to the ms-rest-js in the Model_repository package.json file I believe we need to update that one too.

File: digitaltwins/model_repository/src/pl/models/index.ts (hunk 1 above)
Comment (zolvarga) [reply digitaltwins/model_repository/src/pl/models/index.ts]: This is generated code

---
In reply to: [371559130](https://github.com/Azure/azure-iot-sdk-node/pull/681#discussion_r371559130) [](ancestors = 371559130)

**Comment by zolvarga** [comment]
I did it, it is in this PR :)

---
In reply to: [579026743](https://github.com/Azure/azure-iot-sdk-node/pull/681#issuecomment-579026743) [](ancestors = 579026743)

**Comment by zolvarga** [comment]
Thanks for the review!

---
In reply to: [349073245](https://github.com/Azure/azure-iot-sdk-node/pull/681#issuecomment-349073245) [](ancestors = 349073245)

**Pull request merged**